config/locales/en.yml

```diff
@@ -902,6 +902,7 @@ en:
             other: "%{count} products could not be saved. Please review the errors and try again."
         reset: Discard changes
         save: Save changes
+        new_variant: New variant
```

Comment:
The `new_variant` string was moved from _table to _product_variant_row (below), so I don't see why this is needed here, can you please double-check?

Reply:
I thought I checked that already, but I just found some duplicate code probably a rebase error, I am investigating